games-arcade/cavezofphear/cavezofphear-0.5.1.ebuild

```diff
@@ -1,13 +1,12 @@
-# Copyright 1999-2014 Gentoo Foundation
+# Copyright 1999-2019 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI=5
 inherit eutils games
 
 DESCRIPTION="A boulder dash / digger-like game for console using ncurses"
 HOMEPAGE="http://www.x86.no/cavezofphear/"
-SRC_URI="http://www.x86.no/${PN}/${P/cavezof}.tar.bz2"
-
+SRC_URI="https://mirror.amdmi3.ru/distfiles/phear-${PV}.tar.bz2"
```

Comment:
This is a Gentoo mirror, which means this file is available on all Gentoo mirrors. Put this instead.

```bash
SRC_URI="mirror://gentoo/phear-${PV}.tar.bz2"
```